kyo-core/shared/src/test/scala/kyo/AsyncTest.scala

```diff
@@ -756,7 +756,7 @@ class AsyncTest extends Test:
                             f(1),
                             f(1)
                         ),
-                        2
+                        3
```

Comment:
unrelated flaky test fix. The computations should always run in parallel in order for the memo count call to always be 3.